packages/components/data-table/src/header-cell.styles.tsx

```diff
@@ -123,8 +123,8 @@ const HeaderIconWrapper = styled.div`
   display: inline-flex;
   align-items: center;
   justify-content: center;
-  margin-left: ${vars.spacingS};
-  padding: 0 ${vars.spacingXs};
+  margin: 0 ${vars.spacingS};
```

Comment:
I think the margin should be left only. When the cell content is right-aligned it's visible that the space on the right is bigger than normal.

<img width="190" alt="image" src="https://user-images.githubusercontent.com/1110551/192475713-b6441cca-93d1-4017-9a58-3d22b29c2249.png">

Reply:
Thank you. 
Updated here: https://github.com/commercetools/ui-kit/pull/2288/commits/5d18b00a597a722dbaf8b1b4eea7dd20f1ae9ba8